admin-dev/themes/default/template/helpers/list/list_header.tpl

```diff
@@ -59,12 +59,12 @@
 {/if}
 {* Display column names and arrows for ordering (ASC, DESC) *}
 {if $is_order_position}
-	<script type="text/javascript" src="../js/jquery/plugins/jquery.tablednd.js"></script>
+	<script type="text/javascript" src="{$js_dir}jquery/plugins/jquery.tablednd.js"></script>
```

Comment:
```suggestion
	<script type="text/javascript" src="{$smarty.const._PS_JS_DIR_}jquery/plugins/jquery.tablednd.js"></script>
```

Reply:
I can't do this because, with the horizontal migration, there is no `smarty` variable. So I need to add a new variable like `js_dir`